src/backend/marsha/core/migrations/0031_auto_20211220_1203.py

```diff
@@ -0,0 +1,82 @@
+# Generated by Django 3.2.10 on 2021-12-20 12:03
+import uuid
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+def migrate_anonumous(apps, schema_editor):
+    """Update existing records to set anonymous_id to a default UUID."""
+    Liveregistrations = apps.get_model("core", "Liveregistration")
+    Liveregistrations.objects.filter(
+        anonymous_id__isnull=True, consumer_site__isnull=True
+    ).update(anonymous_id=uuid.uuid4())
```

Comment:
They will all have the same `uuid` ? Is it what you want ? IMO you can have errors with the constraint `liveregistration_unique_video_anonymous_id`